src/common/hooks/useCategorizeRecentActivity.ts

```diff
@@ -19,7 +19,7 @@ export function useCategorizeRecentActivity() {
       allRecentActivity.reduce<[string[], string[]]>(
         (acc, activity) => {
           // Not order transactions (wrap, approve, etc.) doesn't have class property
-          if (!activity.class || activity.class !== OrderClass.LIMIT) {
+          if (!activity.class || activity.class === OrderClass.MARKET) {
```

Comment:
the comment above reads a bit strange, i get we want for recent history here MARKET orders and TX, but is expressed a bit strangely

Reply:
I would rephrase it like:
// Only display regular onchain transactions (wrap, approval, etc) OR MARKET orders